localstripe/localstripe-v3.js

```diff
@@ -151,17 +164,25 @@ Stripe = (apiKey) => {
   return {
     elements: () => {
       return {
-        create: (type, options) => {
+        create: function(type, options) {
```

Comment:
Why going back to the 2nd millennium? :wink: 
(by the way you add an arrow function on line 382).

Reply:
Because I need to access `this.cardElement`. [According to MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), an arrow function "Does not have its own bindings to `this` or `super`, and should not be used as methods."

Reply:
> Because I need to access `this.cardElement`.

OK!

> You can blame the JavaScript spec for being inconsistent.

;)